srcpkgs/eclipse/template

```diff
@@ -1,27 +1,39 @@
 # Template file for 'eclipse'
 pkgname=eclipse
-version=4.30.0
-revision=2
-#code name of version
-_release=2023-12
-archs="x86_64"
-depends="openjdk11 gtk+3 libwebkit2gtk41 libXtst
+version=4.32.0
+revision=1
+_release="2024-06"
+_jna_version="5.14.0.v20231211-1200"
+archs="x86_64* aarch64*"
```

Comment:
does this really work on musl?

Reply:
Not sure, that was an oversight on my part. I've removed the wildcards. I also added your change to the distfiles and added a comment to the _jna_version variable for future maintainers.